.github/workflows/pr-build.yml

```
@@ -99,6 +165,33 @@ jobs:
             - Artifact Name: `${{ env.APK_NAME }}`
             - QR Code: ${{ steps.upload_qr.outputs.artifact-url }}
 
+      - name: Generate QR code for MinIO APK URL
+        if: false  # TODO: MinIOテスト確認後に削除
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)